tests/features/call_proposals.feature

```diff
@@ -0,0 +1,164 @@
+@api
+Feature: Call for proposals content creation and editing.
+  In order to have Calls for proposals on the site
+  As an editor
+  I need to be able to create, edit and see the call for proposals items
+
+  @javascript
+  Scenario: Creation of a Call for proposals content through the UI.
+    Given I am logged in as a user with the "create oe_call_proposals content, access content, edit own oe_call_proposals content, view published skos concept entities, manage corporate content entities" permission
+
+    And the following document:
+      | name          | file       |
+      | My Document 1 | sample.pdf |
+
+    And the following images:
+      | name          | file           | alt                            |
+      | Contact image | example_1.jpeg | Contact image alternative text |
+
+    When I visit "the Call for proposals creation page"
+    Then I should see "Single-stage" in the "Deadline model"
```

Comment:
I'm wondering if we should actually have a custom step that checks the options of a radios type field. Maybe as an improvement for later.

Reply:
Let's leave it as an improvement for the future.